src/_stylesheets/stylesheet.scss

```
@@ -0,0 +1 @@
+@use "govuk";
```

**question** Not super familiar with what `@use` does, will we still be able to use GOV.UK Frontend's variables, functions and mixins easily in the site's Sass code?

---

It can, but not as written. As Frontend currently doesn't have modules or `@forward` rules, we needs to explicitly assign Frontend's globals to our global namespace like this:

```suggestion
@use "govuk" as *;
```

Then they can be used the same as they are in Frontend (e.g. `color: govuk-colour("red");`)